package.json

```diff
@@ -64,6 +64,8 @@
     "rimraf": "^2.5.2",
     "tap": "^5.7.0",
     "temp": "^0.8.3",
-    "through2": "^2.0.1"
+    "through2": "^2.0.1",
+    "mendel-browserify": "1.0.0",
+    "mendel-requirify": "1.0.0"
```

Comment:
I don't think those are `devDependencies`, right? When someone "npm install mendel -g" we want those to be there, right?